sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2379,31 +2380,49 @@ async Task<Page<DocumentHealthcareResult>> NextPageFunc(string nextLink, int? pa
 
                 try
                 {
-                    int top = 1;
-                    int skip = 1;
+                    int top = default;
+                    int skip = default;
+
+                    // Extracting Job ID and parameters from the URL.
+                    // TODO - Update with Regex for cleaner implementation
+                    // nextLink - https://cognitiveusw2dev.azure-api.net/text/analytics/v3.1-preview.3/entities/health/jobs/8002878d-2e43-4675-ad20-455fe004641b?$skip=20&$top=0
+
                     string[] nextLinkSplit = nextLink.Split('/');
+                    // nextLinkSplit = [ 'https:', '', 'cognitiveusw2dev.azure-api.net', 'text', ..., '8002878d-2e43-4675-ad20-455fe004641b?$skip=20&$top=0']
```

Comment:
if the user passes `showStas` the SDK needs to save that and send it too

Reply:
I imagine it is part of the nextLink so it should be parseable

Reply:
Yes it being sent.